Source/qol/stash.cpp

```diff
@@ -664,6 +630,20 @@ void GoldWithdrawNewText(string_view text)
 	}
 }
 
+void UpdateStashGrid(unsigned page, Point startPoint, Size itemSize, StashStruct::StashCell itemId /*= 0*/)
+{
+	if (itemId == InvalidItemId) {

```

Comment:
This comparison will never be true in the current code. `itemId` is expected to be 1-indexed and defaults to 0, InvalidItemId is expected to be used in comparison with a 0-indexed value.

Reply:
The intent was to create an informal barrier due to an accident that broke things. I accidentally allowed the 65535 value to be written to the grid, which caused all manner of weirdness. I added this to prevent that, admittedly unlikely, situation from occurring again.

That being said, if you feel strongly that is has no value, I am **_not_** arrogant enough to assume I know best and will remove it. :)

Reply:
In that case the following would reflect your intent better:

```suggestion
	if (itemId > stashList.size()) {
```

Reply:
Excellent suggestion yet again! committing